lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -130,27 +131,29 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
     EngineFlutterDisplay.instance,
   ];
 
+  late FlutterViewManager viewManager;
+
   /// Adds [view] to the platform dispatcher's registry of [views].
   void registerView(EngineFlutterView view) {
-    viewData[view.viewId] = view;
+    viewManager.setImplicitView(view);
   }
 
   /// Removes [view] from the platform dispatcher's registry of [views].
   ///
   /// Nothing happens if the view is not already registered.
   void unregisterView(EngineFlutterView view) {
-    viewData.remove(view.viewId);
+    viewManager.unregisterView(view.viewId);
   }
 
   /// The current list of windows.
   @override
-  Iterable<EngineFlutterView> get views => viewData.values;
-  final Map<int, EngineFlutterView> viewData = <int, EngineFlutterView>{};
+  Iterable<EngineFlutterView> get views => viewManager.views;
+  // final Map<int, EngineFlutterView> viewData = <int, EngineFlutterView>{};
```

Comment:
delete?

Reply:
Absolutely, this is superseded by the `FlutterViewManager`